guides/LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright © 2019 Red Hat, Inc.
+Copyright © 2024 Red Hat, Inc.
 
 The text of and illustrations in this document are licensed by Red Hat under a Creative Commons Attribution–Share Alike 3.0 Unported license ("CC-BY-SA"). An explanation of CC-BY-SA is available at http://creativecommons.org/licenses/by-sa/3.0/. In accordance with CC-BY-SA, if you distribute this document or an adaptation of it, you must provide the URL for the original version.
```

Comment:
This does not match the LICENSE file anymore. Is this intentional? Also, lot's of content is probably no longer copyright by Red Hat anymore in "guides/common/modules/" etc.

Reply:
The current text reflects the license in our d/s docs. That's why I only changed the date. I have no idea whether it needs to be the same as the license text for the repo. :-(